openrtb_ext/request.go

```diff
@@ -72,6 +73,9 @@ type ExtRequestPrebid struct {
 	// - basic: excludes debugmessages and analytic_tags from output
 	// any other value or an empty string disables trace output at all.
 	Trace string `json:"trace,omitempty"`
+
+	MultiBid    []*ExtMultiBid         `json:"multibid,omitempty"`
+	MultibidMap map[string]ExtMultiBid `json:"-"`
```

Comment:
Nitpick: Can we make the 'b' in bid uppercase `MultiBidMap`

Reply:
done